python/tvm/contrib/tflite_runtime.py

```diff
@@ -18,7 +18,7 @@
 from .._ffi.function import get_global_func
 from ..rpc import base as rpc_base
 
-def create(tflite_model_bytes, ctx):
+def create(tflite_model_bytes, ctx, target_edgetpu=False):
```

Comment:
instead of a boolean argument, try to use a target string for future expansion: `target='edge_tpu'/'cpu'`

Reply:
thanks for the suggestion, I've made the changes